From 6c83cdb2004708cfc01774b1184e66085201bd04 Mon Sep 17 00:00:00 2001
From: "Arumugam,Kamenee" <kamenee.arumugam@intel.com>
Date: Tue, 31 Mar 2015 15:01:51 +0800
Subject: [PATCH 17/32] iio: accel: lis331dlh: Add header for the LIS331DLH
 driver

This patch is to add the header for the LIS331DLH accelerometer
driver.

Signed-off-by: Tan, Raymond <raymond.tan@intel.com>
Signed-off-by: Pengyu Ma <pengyu.ma@windriver.com>
---
 include/linux/platform_data/lis331dlh_intel_qrk.h |   20 ++++++++++++++++++++
 1 files changed, 20 insertions(+), 0 deletions(-)
 create mode 100644 include/linux/platform_data/lis331dlh_intel_qrk.h

diff --git a/include/linux/platform_data/lis331dlh_intel_qrk.h b/include/linux/platform_data/lis331dlh_intel_qrk.h
new file mode 100644
index 0000000..8d01fe1
--- /dev/null
+++ b/include/linux/platform_data/lis331dlh_intel_qrk.h
@@ -0,0 +1,20 @@
+/*
+ * Platform data for Intel Clanton Hill platform accelerometer driver
+ *
+ * {% copyright %}
+ *
+ */
+
+#ifndef __LINUX_PLATFORM_DATA_LIS331DLH_INTEL_QRK_H__
+#define __LINUX_PLATFORM_DATA_LIS331DLH_INTEL_QRK_H__
+
+/**
+ * struct lis331dlh_intel_qrk_platform_data - Platform data for the ST Micro
+ *                                            accelerometer driver
+ * @irq2_pin: GPIO pin number for the threshold interrupt(INT2).
+ **/
+struct lis331dlh_intel_qrk_platform_data {
+	int irq2_pin;
+};
+
+#endif /* LINUX_PLATFORM_DATA_LIS331DLH_INTEL_QRK_H_ */
-- 
1.7.5.4

